docs/usage/shoot_credentials_rotation.md

```diff
@@ -170,9 +171,50 @@ It is currently **not** rotated automatically and there is no way to trigger it
 
 ## `ServiceAccount` Token Signing Key
 
-This key is used to sign the tokens for `ServiceAccount`s.
-It is currently **not** rotated automatically and there is no way to trigger it manually.
-Note that there is the option to pass such key to Gardener, please see [this document](shoot_serviceaccounts.md#signing-key-secret) for more details.
+Gardener generates a key which is used to sign the tokens for [`ServiceAccount`s](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
```

Comment:
`docs/usage/shoot_serviceaccounts.md` needs update - it says:
```
Note that rotation of this key is not yet supported, hence usage is not recommended.
```

Reply:
I think this is still valid since this speaks about the user-provided signing key. Ref https://github.com/gardener/gardener/pull/5968#discussion_r876012279